tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/Contoso_App.cs

```diff
@@ -0,0 +1,133 @@
+// File generated via LUISGen ..\..\..\..\LUISGenTest\Contoso App.json -c Luis.Contoso_App
```

Comment:
For your code-gen, please follow the same format AutoRest uses:

(Note: I quite like that you included the command line here!) 

```
// <auto-generated>
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for
// license information.
//
// Code generated by Microsoft (R) AutoRest Code Generator.
// Changes may cause incorrect behavior and will be lost if the code is
// regenerated.
// </auto-generated>
```